scripts/dataset_processing/get_commonvoice_data.py

```diff
@@ -88,9 +88,11 @@ def create_manifest(data: List[tuple], output_name: str, manifest_path: str):
 
     with output_file.open(mode='w') as f:
         for wav_path, duration, text in tqdm(data, total=len(data)):
-            f.write(
-                json.dumps({'audio_filepath': os.path.abspath(wav_path), "duration": duration, 'text': text}) + '\n'
-            )
+            if wav_path != '':
+                # skip invalid input files that could not be converted
```

Comment:
If skipping, log that file xyz could not be passed / added to manifest.

Reply:
that is done.